docs/MAINTAINING.md

```diff
@@ -36,6 +37,18 @@ Here are the milestones we use and how we use them:
 
 With this flow, any un-assigned issues are essentially in triage state and once triaged are either "Backlog" or "Backlog Candidates". They will eventually move to "On Deck" (or be closed). Lastly, they will end up on a version milestone where they will be worked on.
 
+### Triage
+
+We use the following process for triaging GitHub issues:
+
+1. issue comes in
```

Comment:
Oh yeah `1.1.1.` - habits 😅 I'll adjust!

Reply:
Haha I gotta admit I will love you if you number them because I only read markdown in my editor and the 11111 doesn't look great.

Ideally your editor would re-number them for you but I'm not sure if VS Code has that functionality or if there's a plugin for it.

Reply:
@code-asher I guess that you don't use VSCode's built-in Markdown Preview?

Reply:
Nope, I only read markdown raw.

Reply:
+1 with asher; I usually read markdown in gnome-text-editor because native editors are just plain faster than the fancy preview magic in my experience.